pkg/accumulator/accumulator_interface.go

```diff
@@ -0,0 +1,7 @@
+package accumulator
+
+type AcccumulatorClient interface {
+	StartCacheAccumulator() error
```

Comment:
[Question]: Do you mean `StartAccumulatorCache`? What does this function do? please add some comments